docs/user_guide/datasets/adult_data.rst

```diff
@@ -0,0 +1,18 @@
+.. _retiring-adult-data:
+Retiring Adult
+-------------------------
+
+This section contains datasets created by Ding et al. [1]_, which are described
+in their published paper "Retiring Adult: New Datasets for Fair Machine Learning" (2021).
```

Comment:
Shall we make this `_adult_data`, header `Adult census dataset`, and then mention that the adult dataset is very popular benchmark for algorithmic interventions w.r.t. fairness metrics (like you did in the other file)?
After that as a brief intro, the current lines 5&6 are great since they give us an idea of what to expect in the other files.

Reply:
I guess a citation for the adult dataset wouldn't hurt in that case either. You can find it in the API ref of fetch_adult

Reply:
Updated the cross-reference tag, header, and added the citation for UCI Adult